src/app/features/busSearchSlice.ts

```diff
@@ -0,0 +1,67 @@
+import { createSlice } from '@reduxjs/toolkit';
+import type { PayloadAction } from '@reduxjs/toolkit';
+import { IBusType, ISeatType } from '../../types';
+
+interface IBusSearchParams {
+    originID?: number;
+    destinationID?: number;
+    tripDate?: string;
+    sortBy?: string;
+    sortOrder?: string;
+    seatType?: ISeatType | null;
```

Comment:
null needed?

Reply:
Was added when there was a case with filters being null, now it that case has been handled and can be removed. Rectified issue in commit: https://github.com/nijin-26/bus-ticket-booking-react/pull/24/commits/949f9eb00945de8723e70618e8816603f116682a